docs/presentation/Content/index.md

```diff
@@ -1,35 +1,37 @@
-# Content-related Rules
+# Content
 
-There are 2 primary concepts for creating content: rules and specifications.
+The Presentation library provides a declarative way to create content for tables, property grid and other content components based on iModel data.
 
-## Rules
+![Content rules' example](./media/content-rules-example.png)
 
```

Comment:
None of the images show up because they reside in the wrong folder.

Reply:
fixed